tests/date/SafeDate.test.ts

```diff
@@ -1,62 +1,70 @@
-import { Decoder, isSafeDate, Eq, Guard, TaskDecoder, Type } from '../src/date/SafeDate'
+import * as SafeDate from '../../src/date/SafeDate'
```

Comment:
I mostly did this so I could use an easy snippet to add all the `Arbitrary` tests, but also I think it just makes more sense in these files